app/controllers/events/view/sessions/list.js

```diff
@@ -161,6 +169,35 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async updateRating(session, rating) {
```

Comment:
How are you handling failure cases ? (like when `save` or `createRecord`) fails ?

Reply:
@niranjan94 I've added `try-catch` for handling error.